src/services/utils.ts

```diff
@@ -12,11 +12,52 @@ export const backendFetch = (url: string, init: any, token?: string) => {
     return safeFetch(url, initCopy);
 };
 
-export const backendFetchJson = (url: string, init: any, token?: string) => {
-    const initCopy = prepareRequest(init, token);
-    return safeFetch(url, initCopy).then((safeResponse) =>
-        safeResponse.status === 204 ? null : safeResponse.json()
+export async function backendFetchJson(
+    url: string,
+    init?: any,
+    token?: string
+) {
```

Comment:
@TheMaskedTurtle what do you think that we do
```suggestion
): Promise<unknown> {
```
to force a check or explicit cast where it's use?

Reply:
I like this solution https://stackoverflow.com/questions/41103360/how-to-use-fetch-in-typescript

Reply:
@Meklo
My remark wasn't for `backendFetch` but for `backendFetchText` + `backendFetchJson` + `backendFetchBlob` 😓

Reply:
Doesn't `backendFetch` also need to be parametrized tho ? Otherwise how can we cast `Promise<Response>` to the desired type, see functions in study service